deployment/ansible/download_chain.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Playbook used to download latest blockchain data from last host in the playbook hosts
+
+- name: Archive and download epoch blockchain
+  hosts: "{{ ansible_play_hosts | last }}"
+  remote_user: epoch
+
+  vars:
+    project_root: "{{ ansible_env.HOME }}/node"
+    download_dir: /tmp/blockchain
+
+  tasks:
+    - name: Stat epoch.yaml
+      stat:
+        path: "{{ project_root }}/epoch.yaml"
+      register: config_file
+
+    - name: Read epoch.yaml contents
+      command: "cat {{ project_root }}/epoch.yaml"
+      register: cat_output
+      when: config_file.stat.exists
+
+    - name: Parse YAML configuration
+      set_fact:
+        epoch_vars: "{{ cat_output.stdout | from_yaml }}"
+      when: config_file.stat.exists
+
+    - name: Archive blockchain
+      archive:
+        path: "{{ project_root }}/{{ epoch_vars.chain.db_path }}/"
+        dest: "{{ ansible_env.HOME }}/{{ epoch_vars.chain.db_path }}.tgz"
```

Comment:
This path is created on the node the Ansible client runs i.e. the CI executor, right?

Reply:
I mean - if this creates the dest file on the managed node i.e. the UAT node it is not ok.

Reply:
It's on the managed node, what's your concerns ?

Reply:
> It's on the managed node, what's your concerns ?

Path may be already present. What is behaviour then?
If script happens to run on Sun and Tue on same node and db_path is still same in deployment, issue in script somehow may lead to old archive copied.

If file created in managed node, as I now understand it is, may I suggest naming archive timestamp+db_path (or better timestamp+nodename+db_path) rather than only db_path? This would prevent any silent copying of pre-existing archive.

Reply:
> Path may be already present. What is behaviour then?

It overwrites it. Already tested.

Reply:
`db_path` is as the name implies filesystem path, not suitable to be used in the filenames as is.
I prefer to keep ansible playbooks as simple as possible, thus just added a `archive_suffix` variable that can be used for uniqueness.

The changes are in 20bd2918